test/TestUtilities/Test.Utility/CommandRunner.cs

```diff
@@ -116,5 +118,39 @@ private static async Task ConsumeStreamReaderAsync(StreamReader reader, StringBu
                 lines.AppendLine(line);
             }
         }
+
+        private static void Kill(Process process)
+        {
+            try
+            {
+                process.Kill();
```

Comment:
What's the scenario where kill gets an access denied exception?

Reply:
From https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill?view=netframework-4.8:

> ! Note
If the call to the Kill method is made while the process is currently terminating, a Win32Exception is thrown for Access Denied

Reply:
welp :D 
Thanks, I guess I was more asleep thna I thought :D